staff/ferran-calvo/precourse/proyecto1/calculadora.js

```diff
@@ -0,0 +1,84 @@
+let historialOperaciones = [];
+let vecesOperacion = 0;
+
+function suma(numeros){
+    let resultadoSuma = numeros[0];
+    let stringResultado = numeros[0].toString();
+    for (i=1; i < numeros.length; i++){
+        resultadoSuma += numeros[i];
+        stringResultado += `+${numeros[i]}`;
+    }
+    stringResultado += `= ${numeroDecimales(resultadoSuma)}`;
+    return stringResultado;
+}
+function resta(numeros){
+    let resultadoResta = numeros[0];
+    let stringResultado = numeros[0].toString();
+    for (i=1; i < numeros.length; i++){
+        resultadoResta -= numeros[i];
+        stringResultado += `-${numeros[i]}`;
+    }
+    stringResultado += `= ${numeroDecimales(resultadoResta)}`;
+    return stringResultado;
+}
+function multiplicacion(numeros){
+    let resultadoMultiplicacion = numeros[0];
+    let stringResultado = numeros[0].toString();
+    for (i=1; i < numeros.length; i++){
+        resultadoMultiplicacion *= numeros[i];
+        stringResultado += `*${numeros[i]}`;
+    }
+    stringResultado += `= ${numeroDecimales(resultadoMultiplicacion)}`;
+    return stringResultado;
+}
+function division(numeros){
+    let resultadoDivision = numeros[0];
+    let stringResultado = numeros[0].toString();
+    for (i=1; i < numeros.length; i++){
+        resultadoDivision /= numeros[i];
+        stringResultado += `/${numeros[i]}`;
+    }
+    stringResultado += `= ${numeroDecimales(resultadoDivision)}`;
+    return stringResultado;
+}
+function nuevaOperacion(){
+    let pregunta = prompt("¿Desea realizar una operación? y/n");
+    if (pregunta == "y"){
```

Comment:
Hi, in order to have a correct answer from the user, maybe it's a good idea convert to lowerCase the answer variable. Ex: pregunta.toLowerCase() == "y"

Reply:
Thanks, done!